packages/db/src/sql/_createTables.ts

```
@@ -23,63 +23,54 @@ function createTables<T extends string>(
                 // eslint-disable-next-line @typescript-eslint/promise-function-async
                 .then(() =>
                   Promise.all(
-                    ((indexes[table] as string[]) != null
-                      ? (indexes[table] as string[])
-                      : []
-                    ).map<
+                    ((indexes[table] as string[]) !== null ? (indexes[table] as string[]) : []).map<
                       Promise<any>
```

Biome (2.4.9)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)